HW2/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>HW2 FORMS</title>
+    <link rel="stylesheet" type="text/css" href="style/style.css">
+    <meta charset="UTF-8">
+</head>
+<body>
+    <form method="post">
+        <div class="block">
+            ID <input type="text" name="id" value="725" disabled><br>
+            First Name <input type="text" name="firstName"><br>
+            Last Name <input type="text"name="lastName"><br>
+            Password <input type="password" name="password"><br>
+            Phone <input type="text" placeholder="380" name="phone"><br>
+            Email <input type="text" placeholder="email@website.com" name="email"><br>
+            <input type="checkbox" name="emailHidden">Keep my email address private<br>
+            We'll remove your public profile email and use
+            1978331+username@users.noreply.localhost<br>
+            DOB <input type="date" name="dob"><br>
+            Blog url <input type="url" placeholder="http://" name="blogUrl"><br>
+            About me <textarea type="text" placeholder="Tell us smth about you" name="aboutMe" rows="5"></textarea><br>
+            Profile Picture <input type="file" value="choose the file" name="file">
+            <input type="hidden" value="forExampleHereIsInformationAboutUserAgent">
+        </div>
+
+        <div class="block">
+            Desire to study <input type="range" min="0" max="100" step="1"> <br>
+            Experience <input type="number" name="experience" step="0.5" value="1" min="0.5" max="50"> years <br>
+            Development Experience <br>
+            <input type="radio" id="learningToProgram" name="devExp" value="learning">
+            <label for="learningToProgram">Learning to Program</label> <br>
+            <input type="radio" id="juniorDeveloper" name="devExp" value="junior">
+            <label for="juniorDeveloper">Junior Developer</label> <br>
+            <input type="radio" id="midLevelDeveloper" name="devExp" value="midLevel">
+            <label for="midLevelDeveloper">Mid-Level Developer</label> <br>
+            <input type="radio" id="seniorDeveloper" name="devExp" value="senior">
+            <label for="seniorDeveloper">Senior Developer</label> <br>
+            Country <select name="country">
```

Comment:
Preferred choices is missed. See wire-frame

Reply:
repaired this all ;)

Reply:
Can't understand how to make a preffered choise withous using a Symfony or js. May I just add a line (using style.css)??? :)